libs/components/forms/src/lib/modules/input-box/input-box.component.scss

```diff
@@ -394,9 +420,32 @@ sky-input-box {
       }
 
       textarea.sky-form-control {
-        margin-top: 0;
+        margin-top: -23px;
         padding-top: 0;
         resize: vertical;
+        border-top: 26px solid transparent;
+
+        &:autofill,
+        &:-webkit-autofill {
+          border-top: 26px solid transparent !important;
+          box-shadow: 0 0 0 1px $sky-theme-modern-border-color-neutral-medium !important;
```

Comment:
Autofill styles overwrite the box shadow and such - and they use `!important` - so to maintain them - we must do this using `!important`. Adam wanted the borders and everything maintained when using autofill.